src/components/CompareResults/SubtestsResults/SubtestsRevisionRow.tsx

```diff
@@ -122,6 +122,7 @@ function determineSign(baseMedianValue: number, newMedianValue: number) {
 }
 
 function SubtestsRevisionRow(props: RevisionRowProps) {
+  const id = useId();
```

Comment:
you also need to add the aria-expanded etc in this component, like in the other one :-)

Reply:
ah missed that. fixing now